mdm/api/src/main/java/com/gentics/mesh/core/jobs/SingleJobProcessor.java

```diff
@@ -0,0 +1,16 @@
+package com.gentics.mesh.core.jobs;
+
+import com.gentics.mesh.core.data.job.HibJob;
+import io.reactivex.Completable;
+
+/**
+ * A job processor responsible for performing a single job
+ */
+public interface SingleJobProcessor {
```

Comment:
Shouldn't it extend the JobProcessor?

Reply:
I have made a distinction between the two interfaces. `JobProcessor` is actually fetching jobs and passing them to the right `SingleJobProcessor`, and it takes no parameters in its process method.